src/breakpoints.jl

```diff
@@ -224,10 +248,7 @@ end
 Toggle breakpoint `bp`.
 """
 toggle(bp::AbstractBreakpoint) = (bp.enabled[] = !bp.enabled[]; update_states!(bp))
-function toggle(bp::BreakpointRef)
-    state = bp[]
-    bp.framecode.breakpoints[bp.stmtidx] = BreakpointState(!state.isactive, state.condition)
-end
+toggle(bp::BreakpointRef) = update_state!(bp, !bp[].isactive)
```

Comment:
This doesn't seem equivalent to the previous code?

Reply:
I think it is.
substituting in `update_state!`
This is equie to `bp[] = !bp[].isactive`

Now we look with those do
https://github.com/JuliaDebug/JuliaInterpreter.jl/blob/08333287554a7aa54df447ee13dd465c87afef8b/src/types.jl#L290-L292
Subtitupe in `setindex`
And we get
`bp.framecode.breakpoints[bp.stmtidx] = BreakpointState( !bp[].isactive, bp[].condition)`
extract out the `bp[]` and we get
```
state=bp[]
bp.framecode.breakpoints[bp.stmtidx] = BreakpointState( !state.isactive, state.condition)
```

Which is what was there before.